app/controllers/application_controller.rb

```diff
@@ -4,7 +4,12 @@ def homepage
   end
 
   def profile
-    @user = current_medwallet_account
-    render template: "profiles/home.html.erb"
+    if MedwalletAccount.find_by(id: params[:id]) != nil
```

Comment:
[`nil` is "falsey" in Ruby](https://stackoverflow.com/a/34594746), meaning you can refactor this to

```suggestion
    if MedwalletAccount.find_by(id: params[:id])
```